python/cugraph/cugraph/centrality/betweenness_centrality.py

```diff
@@ -278,9 +276,12 @@ def edge_betweenness_centrality(
         df['dst'] : cudf.Series
             Contains the vertex identifiers of the destination of each edge
 
-        df['edge_betweenness_centrality'] : cudf.Series
+        df['betweenness_centrality'] : cudf.Series
```

Comment:
> Is this being changed because the old description was incorrect

Yes because the description was incorrect. Both the cython and the python code returned a column named "betweenness_centrality"